### PR TITLE
trigger api

### PR DESCRIPTION
**Pull request by helenaford**

### CODE REVIEW

File: src/types/Module.ts [hunk 1]
```diff
@@ -27,6 +28,31 @@ export interface Module {
...
+   * notifications.
+   *
+   * Currently only supported on Android.
+   *
+   * @platform android
+   */
+  cancelDeliveredNotifications(): Promise<void>;
+
+  /**
+   * API used to cancel all scheduled notifications.
```
Comment (mikehardy) [inline src/types/Module.ts:46]: I know this is draft, and I have nothing actually constructive to add, but I was reading it out of curiosity and noticed copy-paste docs which are a classic thing for me to forget later so I thought I'd drop a note

File: src/types/Trigger.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+   *
+   * Must be set to a minimum of 15 minutes.
+   *
+   */
+  repeatInterval?: number;
+
+  /**
+   * The unit of time that the `repeatInterval` is measured in.
+   *
+   * For example, if set to `TimeUnit.DAYS` and repeat interval is set to 3, the notificaiton will repeat every 3 days.
```
Comment (mikehardy) [inline src/types/Trigger.ts:35]:
```suggestion
   * For example, if set to `TimeUnit.DAYS` and repeat interval is set to 3, the notification will repeat every 3 days.
```

slightly more useful, but only slightly, a typo spotted

File: src/types/Trigger.ts [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+   * The timestamp when the notification should first be shown, in milliseconds since 1970.
+   */
+  timestamp: number;
+
+  /**
+   * How frequently the notification should be repeated.
+   *
+   * If not present, the notification will be displayed once.
+   *
+   * For example, if set to 30, the notificaiton will be displayed every 30 minutes.
```
Comment (mikehardy) [inline src/types/Trigger.ts:25]:
```suggestion
   * For example, if set to 30, the notification will be displayed every 30 minutes.
```

File: src/types/Module.ts (hunk 1 above)
Comment (helenaford) [reply src/types/Module.ts]: thanks Mike, appreciate the review 👍

File: src/types/Notification.ts [hunk 4]
```diff
@@ -267,6 +272,7 @@ export interface EventDetail {
    *  - [`EventType.PRESS`](/react-native/reference/eventtype#press)
    *  - [`EventType.ACTION_PRESS`](/react-native/reference/eventtype#action_press)
    *  - [`EventType.DELIVERED`](/react-native/reference/eventtype#delivered)
+   *  - [`EventType.SCHEDULED`](/react-native/reference/eventtype#scheduled)
```
Comment (Salakar) [inline src/types/Notification.ts:275]: Should this be `TRIGGER_CREATED`?

Comment (helenaford) [reply src/types/Notification.ts:275]: yes, nice spot!!

**Pull request merged**